src/sub_8001A06C.c

```diff
@@ -0,0 +1,18 @@
+#include "linker.h"
+static int SECTION(".bss_800B1E08") strangeTable_800B1E08[16];
+int sub_8001A06C(void)
+{
+    int *tbl_p;
+    int it;
+
+    tbl_p = (int*) &strangeTable_800B1E08;
+    it = 8;
+    for (it = 8; 0 < it;)
+    {
+        it -= 1;
```

Comment:
should probably go in the for after the ;

Reply:
Tried, non match. "after the ;" would happen after the other commands in the loop